app/views/profile/friends/index.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1>FRIENDS SKELETON PLACEHOLDER</h1>
+
+<% if @friends == [] %>
```

Comment:
This is really the only thing that stands out to me, I'm surprised rubocop doesn't ding this

Reply:
same